### ISSUE TITLE
Support for Hyprland!

### ISSUE THREAD

**Issue by ghost** [NomisIV/swayhide#10]
Support for Hyprland, PLEASE!

**Comment by rehanzo** [comment]
sure, I assume you see it working the same way as it works in sway, just with the default special workspace in place of the scratchpad?

**Comment by NomisIV** [comment]
Hyprland already has native window swallowing: Look for `enable_swallow` in [this table](https://wiki.hyprland.org/Configuring/Variables/#misc)

**Comment by rehanzo** [comment]
> Hyprland already has native window swallowing: Look for `enable_swallow` in [this table](https://wiki.hyprland.org/Configuring/Variables/#misc)

right, but one might want the parent window still open, but just stowed away, and able to be accessed without the child being killed.
